test/test_models_detection_anchor_utils.py

```diff
@@ -85,5 +86,5 @@ def test_defaultbox_generator(self):
         self.assertEqual(len(dboxes), 2)
         self.assertEqual(tuple(dboxes[0].shape), (4, 4))
         self.assertEqual(tuple(dboxes[1].shape), (4, 4))
-        self.assertTrue(dboxes[0].allclose(dboxes_output))
-        self.assertTrue(dboxes[1].allclose(dboxes_output))
+        torch.testing.assert_close(dboxes[0], dboxes_output, rtol=1e-5, atol=1e-8)
+        torch.testing.assert_close(dboxes[1], dboxes_output, rtol=1e-5, atol=1e-8)
```

Comment:
Here it's a bit of a shame IMHO that we're forced to specify `atol` as well. The default for `atol` is 1e-5 and 
```py
        torch.testing.assert_close(dboxes[0], dboxes_output, rtol=1e-5, atol=1e-5)
        torch.testing.assert_close(dboxes[1], dboxes_output, rtol=1e-5, atol=1e-5)
```
passes just fine, so ideally we could just write

```py
        torch.testing.assert_close(dboxes[0], dboxes_output, rtol=1e-5)
        torch.testing.assert_close(dboxes[1], dboxes_output, rtol=1e-5)
```